pkg/filesystem/subvolume.go

```diff
@@ -34,10 +33,17 @@ type fsStruct struct {
 }
 
 type subVolumeInfo struct {
-	svg string
-	fs  string
+	svg   string
+	fs    string
+	state string
 }
 
+const (
+	InUse            = "In-Use"
+	Stale            = "Stale"
+	StaleAndSnapshot = "Stale And Has Snapshot"
```

Comment:
If this is printed as a column, I would suggest it's nice to have a single word to make the output more parsable
```suggestion
	StaleAndSnapshot = "Stale-with-Snapshot"
```